packages/@glimmer/integration-tests/test/strict-mode-test.ts

```diff
@@ -646,6 +737,19 @@ class DynamicStrictModeTest extends RenderTest {
     this.assertStableRerender();
   }
 
+  @test({ skip: true })
+  'Calling a dynamic modifier without a value is a no-op'() {
+    const Bar = defineComponent(
+      {},
+      '<div {{this.foo 123}}></div>',
+      class extends GlimmerishComponent {}
+    );
+
+    this.renderComponent(Bar);
+    this.assertHTML('');
+    this.assertStableRerender();
+  }
```

Comment:
This is just future work, right?

Reply:
Yes, that's why the test was skipped, was just easier to write it out now while I had the context